drud/ddev
# Upgrade containers (and mariadb to 10.2), fixes #947, fixes #1068

## The Problem/Issue/Bug:

* ddev-webserver needs to go to current stable debian stretch, fixing #947 
* ddev-dbserver goes to alpine:3.8
* mariadb version goes to 10.2, fixing #1068
* Bump versions and update sources for all containers.

## How this PR Solves The Problem:

## Manual Testing Instructions:

Try it out, it should work.
Preferably use the full circleci build so you get all 3 new images. 
Risks include doing a restore-snapshot (onto mariadb 10.2) on a previous snapshot (Created on mariadb 10.1). So try that.  Also *starting* an existing 10.1 project with this.

## Automated Testing Overview:


## Related Issue Link(s):

#947 was about upgrading the db container to mariadb 10.2
#1068 is about upgrading the web container to debian stable.

## Release/Deployment notes:
